tests/integration/test_logcollector/test_location/data/configuration/wazuh_location.yaml

```diff
@@ -0,0 +1,26 @@
+---
+# conf 1

```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/705ee88023c33aee10ab5393267a9bb23c335048